reference/3.0/Microsoft.PowerShell.Management/Get-Process.md

```diff
@@ -7,126 +7,175 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=113324
 
-WorkingSet is one of many properties of process objects.
-To see all of the properties, type "Get-Process | Get-Member".
-By default, the values of all amount properties are in bytes, even though the default display lists them in kilobytes and megabytes.
-### Example 4
-```
-PS C:\> $a = get-process PS C:\> get-process -inputobject $a | format-table -view priority
+**WorkingSet** is one of many properties of process objects.
+To see all of the properties, type `Get-Process | Get-Member`.
+By default, the values of all amount properties are in bytes, even though the default display lists
+them in kilobytes and megabytes.
+
+### Example 4: List processes on the computer in groups based on priority
+
+```powershell
+$A = Get-Process
+$A | Get-Process | Format-Table -View priority
 ```
 
-These  commands list the processes on the computer in groups based on their priority class.
-The first command gets all the processes on the computer and then stores them in the $a variable.
+These commands list the processes on the computer in groups based on their priority class.
+The first command gets all the processes on the computer and then stores them in the `$A` variable.
 
-The second command uses the InputObject parameter to pass the process objects that are stored in the $a variable to the Get-Process cmdlet.
-The pipeline operator passes the objects to the Format-Table cmdlet, which formats the processes by using the Priority view.
-The Priority view, and other views, are defined in the PS1XML format files in the Windows PowerShell home directory ($pshome).
+The second command pipes the **Process** object stored in the `$A` variable to the
+`Get-Process` cmdlet, then to the `Format-Table` cmdlet, which formats the processes
+by using the **Priority** view.
+
+The **Priority** view, and other views, are defined in the PS1XML format files in the PowerShell
+home directory (`$pshome`).
+
+### Example 5: Add a property to the standard Get-Process output display
 
-### Example 5
 ```powershell
-PS C:\> Get-Process powershell -ComputerName S1, localhost |
-	ft @{Label = "NPM(K)"; Expression = {[int]($_.NPM / 1024)}},
-	@{Label = "PM(K)"; Expression = {[int]($_.PM / 1024)}},
-	@{Label = "WS(K)"; Expression = {[int]($_.WS / 1024)}},
-	@{Label = "VM(M)"; Expression = {[int]($_.VM / 1MB)}},
-	@{Label = "CPU(s)"; Expression = {if ($_.CPU) {$_.CPU.ToString("N")}}},
-	Id, MachineName, ProcessName -Auto
+Get-Process powershell -ComputerName S1, localhost | Format-Table `
+                               @{Label = "NPM(K)"; Expression = {[int]($_.NPM / 1024)}},
+                               @{Label = "PM(K)"; Expression = {[int]($_.PM / 1024)}},
+                               @{Label = "WS(K)"; Expression = {[int]($_.WS / 1024)}},
+                               @{Label = "VM(M)"; Expression = {[int]($_.VM / 1MB)}},
+                               @{Label = "CPU(s)"; Expression = {if ($_.CPU) {$_.CPU.ToString("N")}}},
```

Comment:
Why so many spaces? And the last line does not line up. Change it to: 

```
Get-Process powershell -ComputerName S1, localhost | Format-Table `
    @{Label = "NPM(K)"; Expression = {[int]($_.NPM / 1024)}},
    @{Label = "PM(K)"; Expression = {[int]($_.PM / 1024)}},
    @{Label = "WS(K)"; Expression = {[int]($_.WS / 1024)}},
    @{Label = "VM(M)"; Expression = {[int]($_.VM / 1MB)}},
    @{Label = "CPU(s)"; Expression = {if ($_.CPU) {$_.CPU.ToString("N")}}},
    Id, MachineName, ProcessName -AutoSize
```

Make this change in all versions.

Reply:
Done